crates/viewer/re_renderer/shader/world_grid.wgsl

```diff
@@ -0,0 +1,154 @@
+#import <./global_bindings.wgsl>
+
+struct WorldGridUniformBuffer {
+    color: vec4f,
+
+    /// A value of [`super::GridPlane`]
+    orientation: u32,
```

Comment:
Come to think of it; it would be better if we could fully specify the plane using the plane equation. That would allow us to use this shader with a general `Plane3D` primitive in Rust

Reply:
hmmm yeah why not I guess. Won't expose it in the viewer though this way since imho there we want a simple interface for the view properties

Reply:
actually... maybe if we have a ui to pick the axis plane easily, then why not all the way hmmmm. Undecided. To be dicussed on the next pr